### ISSUE TITLE
Visual Studio Test Agent Deployment task fails with error 1603

### ISSUE THREAD

**Issue by FramnkRulez** [Microsoft/vsts-tasks#3407]
Our build configuration has the 'Visual Studio Test Agent Deployment' task included for running automated tests.  However this task is failing with error 1603 on a clean machine.  Attached is the installation log I found in %temp%

[dd_vstf_testagent_20170111102422.txt](https://github.com/Microsoft/vsts-tasks/files/699319/dd_vstf_testagent_20170111102422.txt)

**Comment by tanvi-soni** [comment]
Can you please send the build logs as well to devops_tools@microsoft.com

**Comment by FramnkRulez** [comment]
I figured out the problem with this issue.  The Visual Studio Test Agent Deployment tasks fails on a machine without an active internet connection.  The failure is due to it trying to validate the root certificate for authenticode and then later in the install when it attempts to check the certificate revocation list for the signed installation content.

I had to enable my internet connection on the virtual machine I was attempting to deploy the agent to.
